host_vars/helium.collegiumv.org.yml

```diff
@@ -5,4 +5,3 @@ internal_ip: 192.168.42.2
 peer_internal_ip: 192.168.42.1
 pf_ip: 172.16.0.2
 advskew: 20
-dhcpd_range: 192.168.42.192 192.168.42.254
```

Comment:
Why are we removing this variable instead of merging it (possibly in a group_var)?